appserver/node-express/lib/db-client/contributor/index.js

```diff
@@ -24,6 +27,7 @@ funcs.patchReputation = function (txid, id, repChange) {
   ).result();
 };
 
+// TODO I don't see patchVoteCount() called anywhere in /lib. Can we delete?
```

Comment:
Method is not used anywhere and its functionality is handled in votePatch() in patch.js. I will remove.